deploy/template.yaml

```diff
@@ -209,33 +209,39 @@ Resources:
-              CORE_STUB_PASSWORD: "/perfTest/coreStubPassword"
-              CORE_URL: "/perfTest/coreUrl"
-              ORCH_STUB_URL: "/perfTest/orchStubUrl"
-              RP_STUB: "/perfTest/rpStub"
-              BASE_URL: "/perfTest/baseUrl"
-              F2F_URL: "/perfTest/f2fUrl"
-              K6_DYNATRACE_URL: "/perfTest/dynatraceUrl"
-              HOME_URL: "/perfTest/homeUrl"
-              SIGNIN_URL: "/perfTest/signInUrl"
-              EMAIL_OTP: "/perfTest/fixedEmailOTP"
-              PHONE_OTP: "/perfTest/fixedPhoneOTP"
-              CURR_PHONE: "/perfTest/currPhone"
-              NEW_PHONE: "/perfTest/newPhone"
+              ACCOUNTS_APP_PASSWORD: "/perfTest/account/testUserPassword"
+              ACCOUNTS_APP_PASSWORD_NEW: "/perfTest/account/testUserNewPassword"
+              ACCOUNTS_APP_KEY: "/perfTest/account/accounts/appKey"
+              ACCOUNTS_AUTH_APP_KEY: "/perfTest/account/accounts/authAppKey"
+              ACCOUNTS_BASE_URL: "/perfTest/account/authentication/baseUrl"
+              ACCOUNTS_CURR_PHONE: "/perfTest/account/accounts/currPhone"
+              ACCOUNTS_EMAIL_OTP: "/perfTest/account/fixedEmailOTP"
+              ACCOUNTS_HOME_URL: "/perfTest/account/accounts/homeUrl"
+              ACCOUNTS_NEW_PHONE: "/perfTest/account/accounts/newPhone"
+              ACCOUNTS_PHONE_OTP: "/perfTest/account/fixedPhoneOTP"
+              ACCOUNTS_SIGNIN_URL: "/perfTest/account/accounts/signInUrl"
+              ACCOUNTS_RP_STUB: "/perfTest/account/authentication/rpStub"
+              IDENTITY_ADDRESS_URL: "/perfTest/identity/orange/addressUrl"
+              IDENTITY_ADDRESS_ENV_NAME: "/perfTest/identity/orange/addressEnvName"
+              IDENTITY_CORE_STUB_URL: "/perfTest/identity/coreStubUrl"
+              IDENTITY_CORE_URL: "/perfTest/identity/coreUrl"
+              IDENTITY_DRIVING_ENDPOINT: "/perfTest/identity/lime/drivingEndpoint"
+              IDENTITY_DRIVING_URL: "/perfTest/identity/lime/drivingUrl"
+              IDENTITY_FRAUD_URL: "/perfTest/identity/lime/fraudUrl"
+              IDENTITY_KBV_URL: "/perfTest/identity/orange/kbvUrl"
+              IDENTITY_KBV_ENV_NAME: "/perfTest/identity/orange/kbvEnvName"
+              IDENTITY_KIWI_URL: "/perfTest/identity/kiwi/url"
+              IDENTITY_ORCH_STUB_URL: "/perfTest/identity/orchStubUrl"
+              IDENTITY_PASSPORT_URL: "/perfTest/identity/lime/passportUrl"
+              IDENTITY_STUB_USERNAME: "/perfTest/identity/coreStubUsername"
+              IDENTITY_STUB_PASSWORD: "/perfTest/identity/coreStubPassword"
               K6_DYNATRACE_APITOKEN: "/perfTest/dynatraceApiToken"
+              K6_DYNATRACE_URL: "/perfTest/dynatraceUrl"
+              MOBILE_BACKEND_URL: "/perfTest/mobile/backendUrl"
+              MOBILE_BIOMETRIC_SESSION_ID: "/perfTest/mobile/biometricSessionID"
+              MOBILE_FRONTEND_URL: "/perfTest/mobile/frontendUrl"
+              MOBILE_TEST_CLIENT_EXECUTE_URL: "/perfTest/mobile/testClientUrl"
```

Comment:
```suggestion
              ACCOUNTS_APP_KEY: "/perfTest/account/accounts/appKey"
              ACCOUNTS_APP_PASSWORD_NEW: "/perfTest/account/testUserNewPassword"
              ACCOUNTS_APP_PASSWORD: "/perfTest/account/testUserPassword"
              ACCOUNTS_AUTH_APP_KEY: "/perfTest/account/accounts/authAppKey"
              ACCOUNTS_BASE_URL: "/perfTest/account/authentication/baseUrl"
              ACCOUNTS_CURR_PHONE: "/perfTest/account/accounts/currPhone"
              ACCOUNTS_EMAIL_OTP: "/perfTest/account/fixedEmailOTP"
              ACCOUNTS_HOME_URL: "/perfTest/account/accounts/homeUrl"
              ACCOUNTS_NEW_PHONE: "/perfTest/account/accounts/newPhone"
              ACCOUNTS_PHONE_OTP: "/perfTest/account/fixedPhoneOTP"
              ACCOUNTS_RP_STUB: "/perfTest/account/authentication/rpStub"
              ACCOUNTS_SIGNIN_URL: "/perfTest/account/accounts/signInUrl"
              IDENTITY_ADDRESS_ENV_NAME: "/perfTest/identity/orange/addressEnvName"
              IDENTITY_ADDRESS_URL: "/perfTest/identity/orange/addressUrl"
              IDENTITY_CORE_STUB_URL: "/perfTest/identity/coreStubUrl"
              IDENTITY_CORE_URL: "/perfTest/identity/coreUrl"
              IDENTITY_DRIVING_ENDPOINT: "/perfTest/identity/lime/drivingEndpoint"
              IDENTITY_DRIVING_URL: "/perfTest/identity/lime/drivingUrl"
              IDENTITY_FRAUD_URL: "/perfTest/identity/lime/fraudUrl"
              IDENTITY_KBV_ENV_NAME: "/perfTest/identity/orange/kbvEnvName"
              IDENTITY_KBV_URL: "/perfTest/identity/orange/kbvUrl"
              IDENTITY_KIWI_URL: "/perfTest/identity/kiwi/url"
              IDENTITY_ORCH_STUB_URL: "/perfTest/identity/orchStubUrl"
              IDENTITY_PASSPORT_URL: "/perfTest/identity/lime/passportUrl"
              IDENTITY_STUB_PASSWORD: "/perfTest/identity/coreStubPassword"
              IDENTITY_STUB_USERNAME: "/perfTest/identity/coreStubUsername"
              K6_DYNATRACE_APITOKEN: "/perfTest/dynatraceApiToken"
              K6_DYNATRACE_URL: "/perfTest/dynatraceUrl"
              MOBILE_BACKEND_URL: "/perfTest/mobile/backendUrl"
              MOBILE_BIOMETRIC_SESSION_ID: "/perfTest/mobile/biometricSessionID"
              MOBILE_FRONTEND_URL: "/perfTest/mobile/frontendUrl"
              MOBILE_TEST_CLIENT_EXECUTE_URL: "/perfTest/mobile/testClientUrl"
```
Needs some minor updates to be in alphabetical order

Reply:
Alphabetical order from the path or from the name?